docs/src/logical_replication.md

```diff
@@ -0,0 +1,444 @@
+
+!!! Important
+    Since schema definitions are not replicated, the subscriber must have the
+    corresponding tables already defined before data replication begins.
+
+CloudNativePG simplifies subscription management by enabling you to define them
+declaratively using the `Subscription` resource.
+
+!!! Info
+    Please refer to the [API reference](cloudnative-pg.v1.md#postgresql-cnpg-io-v1-Subscription)
+    for the full list of attributes you can define for each `Subscription` object.
+
+Suppose you want to replicate changes from the `publisher` publication on the
+`app` database of the `freddie` cluster (*publisher*) to the `app` database of
+the `king` cluster (*subscriber*). Here's an example of a `Subscription`
+manifest:
+
+```yaml
+apiVersion: postgresql.cnpg.io/v1
+kind: Subscription
+metadata:
+  name: freddie-to-king-subscription
+spec:
+  cluster:
+    name: king
+  dbname: app
+  name: subscriber
+  externalClusterName: freddie
+  publicationName: publisher
+```
+
+In the above example:
+
+- The subscription object is named `freddie-to-king-subscriber` (`metadata.name`).
+- The subscription is created in the `app` database (`spec.dbname`) of the
+  `king` cluster (`spec.cluster.name`), with name `subscriber` (`spec.name`).
+- It connects to the `publisher` publication in the external `freddie` cluster,
+  referenced by `spec.externalClusterName`.
+
+To facilitate this setup, the `freddie` external cluster must be defined in the
+`king` cluster's configuration. Below is an example excerpt showing how to
+define the external cluster in the `king` manifest:
+
+```yaml
+externalClusters:
+  - name: freddie
+    connectionParameters:
+      host: freddie-rw.default.svc
+      user: postgres
+      dbname: app
+```
+
+!!! Info
+    For more details on configuring the `externalClusters` section, see the
+    ["Bootstrap" section](bootstrap.md#the-externalclusters-section) of the
+    documentation.
+
+As you can see, a subscription can connect to any PostgreSQL database
+accessible over the network. This flexibility allows you to seamlessly migrate
+your data into Kubernetes with nearly zero downtime. It’s an excellent option
+for transitioning from various environments, including popular cloud-based
+Database-as-a-Service (DBaaS) platforms.
+
+### Required Fields in the `Subscription` Manifest
+
+The following fields are mandatory for defining a `Subscription` object:
+
+- `metadata.name`: A unique name for the Kubernetes `Subscription` object
+  within its namespace.
+- `spec.cluster.name`: The name of the PostgreSQL cluster where the
+  subscription will be created.
+- `spec.dbname`: The name of the database in which the subscription will be
+  created.
+- `spec.name`: The name of the subscription as it will appear in PostgreSQL.
+- `spec.externalClusterName`: The name of the external cluster, as defined in
+  the `spec.cluster.name` cluster's configuration. This references the
+  publisher database.
```

Comment:
"references the publisher database" IMO this could be confusing.
The externalClusterName references the publisher cluster, while the publisher database is really implicit in the choice of publicationName, no?

Reply:
Answering my own comment: the database name is in the connection parameters.